app/controllers/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+enable :sessions
+
+get '/users/login' do
+  erb :'users/login_user'
+end
+
+
+post '/users/login' do
```

Comment:
a more RESTful convention here would be `post '/sessions'` since the action taking place within the route is creating a new session for our user!